app/VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 3
+VERSION_MINOR = 3
+PATCHLEVEL = 99
+VERSION_TWEAK = 0
+EXTRAVERSION =
```

Comment:
The existing implementation used the project version from CMake:

https://github.com/zephyrproject-rtos/example-application/blob/be3bfb0ec3ced11311e32e3c507cd83527b4f3d1/app/CMakeLists.txt#L10

Now, there exists no relation between the reported version and the CMake project version.

If we are dropping the use of CMake project version, we might as well drop the `VERSION 1.0.0` from `app/CMakeLists.txt`.

Reply:
Changed it to have the same version of 1.0.0